controllers/promise_controller.go

```diff
@@ -51,6 +54,7 @@ type PromiseReconciler struct {
 }
 
 const (
+	kratixPlatformSystemNamespace                      = "kratix-platform-system"
```

Comment:
thos constant is already defined here https://github.com/syntasso/kratix/blob/a43e0b6c12c59e8733ccd58ff4742031c98e5601/api/v1alpha1/groupversion_info.go#L39